tests/api/routers/test_filing_api.py

```diff
@@ -1042,6 +1045,7 @@ async def test_good_sign_filing(
                 action_type=UserActionType.SIGN,
             ),
         )
+        send_email_mock.assert_called_with("Test User", "test@local.host", "test1@cfpb.gov", ANY, ANY)
```

Comment:
I would do something similar to the `assert upsert_mock` so you can check that the confirmation id and timestamp are also correct (especially confirmation ID since that's a very important part of the email going out).